core/src/main/java/org/elasticsearch/index/query/MoreLikeThisQueryBuilder.java

```diff
@@ -148,9 +148,10 @@
             ParseField DOC = new ParseField("doc");
             ParseField FIELDS = new ParseField("fields");
             ParseField PER_FIELD_ANALYZER = new ParseField("per_field_analyzer");
-            ParseField ROUTING = new ParseField("_routing");
```

Comment:
We've mostly been removing these `Field` interfaces as we go. Could you remove it here too?